src/main/java/seedu/address/model/person/PostalCode.java

```diff
@@ -0,0 +1,63 @@
+package seedu.address.model.person;
+
+import static java.util.Objects.requireNonNull;
+
+import seedu.address.commons.exceptions.IllegalValueException;
+
+/**
+ * Represents a Address' postal code in the address book.
+ * Guarantees: immutable; is valid as declared in {@link #isValidPostalCode(String)}
+ */
+public class PostalCode {
+
+    public static final String MESSAGE_POSTAL_CODE_CONSTRAINTS =
+            "Person postal code should contain only a string of 6 digits prepended with 's' or 'S'";
```

Comment:
The phrasing of this message can be improved, maybe something like "A location's postal code should be a string starting with a 'S' followed by 6 digits.